server/user.test.js

```diff
@@ -0,0 +1,119 @@
+const supertest = require("supertest");
+const mongoose = require("mongoose");
+const app = require("./app");
+
+const variables = {
+  userId: null,
+  token: null,
+};
+
+beforeAll(async () => {
+  await mongoose.disconnect();
+  await mongoose.connect("mongodb://localhost:27017/budgetify");
```

Comment:
It's production DB?

Reply:
As far as I understand it, it is not a production DB. It only exists for tests to successfully run. When i ran the test, the database was newly created and was totally empty.

Reply:
`MONGODB_URI="mongodb://localhost:27017/budgetify"`
I see that DB in `.env` is the same as in tests
You have to use test DB, and wipe all data after tests